isis/tests/CameraFixtures.cpp

```diff
@@ -185,7 +185,7 @@ namespace Isis {
     projCubeLabel >> projLabel;
 
     testCube = new Cube();
-    testCube->fromIsd(tempDir.path() + "/default.cub", label, isd, "rw");
+    testCube->fromIsd("tempDir.path() + default.cub", label, isd, "rw");
```

Comment:
This may be the cause for the majority of the test failures since they all use the DefaultCube fixture.

Reply:
@afrigeri This is still an issue and is causing a bunch of tests to fail. Can the change get reverted?

Reply:
reverted! see 4417d062c